packages/button/stories/index.ts

```diff
@@ -15,9 +15,67 @@ import { ifDefined } from '@spectrum-web-components/base/src/directives.js';
 import '../sp-button.js';
 import '@spectrum-web-components/icon/sp-icon.js';
 import '@spectrum-web-components/icons-workflow/icons/sp-icon-help.js';
+import { ButtonTreatments, ButtonVariants } from '../src/Button.js';
 
-interface Properties {
-    variant?: 'cta' | 'overBackground' | 'primary' | 'secondary' | 'negative';
+export const args = {
+    disabled: false,
+    variant: 'cta',
+};
+
+export const argTypes = {
+    disabled: {
+        name: 'disabled',
+        type: { name: 'boolean', required: false },
+        description:
+            'Disable this control. It will not receive focus or events.',
+        table: {
+            type: { summary: 'boolean' },
+            defaultValue: { summary: false },
+        },
+        control: {
+            type: 'boolean',
+        },
+    },
+    variant: {
+        name: 'variant',
+        type: { name: 'string', required: false },
+        description: 'The visual variant to apply to the button.',
+        table: {
+            type: { summary: 'string' },
+            defaultValue: { summary: 'cta' },
+        },
+        control: {
+            type: 'inline-radio',
+            options: [
+                'cta',
+                'accent',
+                'primary',
+                'secondary',
+                'negative',
+                'overBackground',
+                'black',
+                'white',
+            ],
+        },
+    },
+    treatment: {
+        name: 'tratment',
```

Comment:
Is this what's causing all the Button/*/Outline/Icon-Size-Overridden stories to use treatment='fill'?

```suggestion
        name: 'treatment',
```

Reply:
😱 so many time little things to catch! Added that arg to the icon size stories, thanks.